core/src/main/java/jenkins/security/ApiTokenProperty.java

```diff
@@ -199,7 +199,7 @@ public boolean matchesPassword(String token) {
             return false;
         }
         
-        tokenStats.updateUsageForId(matchingToken.getUuid());
+      //  tokenStats.updateUsageForId(matchingToken.getUuid());
```

Comment:
As a general rule, code shouldn't be commented out, it should either exist or be removed.

That said, I really don't understand your explanation. If you believe something else is doing this already, it'd simplify reviewers work if you provided a stack trace showing that.

Reply:
I filed a bug for this [#JENKINS-56499](https://issues.jenkins-ci.org/browse/JENKINS-56499)